CMakeLists.txt

```diff
@@ -40,7 +40,11 @@ file(GLOB UOSM_CORE_SOURCES
         UOSM-Core/Utils/*.cpp
 )
 
-add_executable(UOSM_Telemetry_Pico ${SOURCES} ${UOSM_CORE_SOURCES})
+add_executable(UOSM_Telemetry_Pico ${SOURCES} ${UOSM_CORE_SOURCES}
+        Core/Tasks/CurrentVoltageTask.h
+        Core/Tasks/CurrentVoltageTask.cpp
```

Comment:
We're working in C, not C++

If there's only C code in the file, it's as easy as renaming it to a .c